src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -308,6 +311,10 @@ protected virtual void ReportUnassignedOutParameter(ParameterSymbol parameter, C
         public static void Analyze(CSharpCompilation compilation, Symbol member, BoundNode node, DiagnosticBag diagnostics, bool requireOutParamsAssigned = true)
         {
             var walker = new DataFlowPass(compilation, member, node, requireOutParamsAssigned: requireOutParamsAssigned);
+            var localFuncWrites = new LocalFunctionAssignmentPass(compilation,
```

Comment:
If you move this to after you run the LocalFunctionAssignmentPass, you can copy the mapping from variables to bit positions from the local function assignment pass.